rollup.config.js

```diff
@@ -16,6 +17,9 @@ export default [false, true].map((minify) => {
   const plugins = [
     nodeResolve(),
     commonjs(),
+    injectProcessEnv({
```

Comment:
not sure about this workaround, without it this error was in the console when opening `test/test-build.html`:

<img width="414" alt="CleanShot 2022-04-03 at 10 18 21@2x" src="https://user-images.githubusercontent.com/6059356/161418750-fa88fa4b-4c53-4581-9d85-582de8de75ec.png">